Sources/ProjectDescription/RunAction.swift

```diff
@@ -10,6 +10,9 @@ public struct RunAction: Equatable, Codable {
     /// Whether a debugger should be attached to the run process or not.
     public let attachDebugger: Bool
 
+    /// The path of custom lldbinit file.
+    public let customLLDBInitFile: String?
```

Comment:
Would this be better suited as a `Path?` ?

Reply:
@kwridan 

i think it's a clearer approach than what this have now. and You meant ```FilePath```, right?

Reply:
> Would this be better suited as a `Path?` ?

oh, i just realized that I'm using that type in tuist. thank you for this comment.

Reply:
There's a `Path` type that is commonly used in manifest files, which supports a few common features such as relative paths to the current manifest, to the root of the workspace, etc...

https://tuist.github.io/tuist/main/documentation/projectdescription/path

There's a few steps to get it working:

- Using type `Path` in the manifests (`ProjectDescription`)
- Resolving `Path` > `AbsolutePath` using `GeneratorPaths` in the Manifest Mapper
- Converting back to relative path in `SchemeDescriptorGenerator` (e.g. see `storeKitFilePath`) to ensure generated projects remain stable and can work across machines if checked in

Reply:
Thanks Kas! @oozoofrog would you mind opening a follow up PR to fix this? Otherwise, I'll try to do it 🙏

Reply:
@danyf90 @kwridan 

Thanks, i follow up new PR for fix this.

Reply:
fixing with this #5071